webserver/website/src/components/navbar.vue

```diff
@@ -16,29 +16,24 @@
         <div class="collapse navbar-collapse" id="navbarSupportedContent">
           <ul class="navbar-nav me-auto mb-2 mb-lg-0">
             <li class="nav-item">
-              <router-link class="nav-link" to="/">Verbindungssuche</router-link>
```

Comment:
Ich würde des schon Verbindungssuche nennen, sagt mehr aus als nur Verbindungen und ich würde auch nicht direkt runterscrollen, sondern entweder in die Mitte zwischen Input form und Ergebnis oder auf des Input form.
Weil man weiß dann schon, dass die Ergebnisse unten sind, und ich würde jetzt sagen, dass wenn man auf den Knopf drückt eher zum Input form will als zu den Ergebnissen.